mx.py

```diff
@@ -749,7 +749,7 @@ def suite_context_free(func):
     """
     Decorator for commands that don't need a primary suite.
     """
-    _suite_context_free.append(func.__name__)
+    _suite_context_free.append(func.__name__.replace("_", "-"))
```

Comment:
This decorator adds _function name_ to list of _commands_. 
Since python doesn't support function names with `-` character, functions that implement  commands with `-` are named with underscore. 

Without this change all such commands wouldn't be registered as `suite_context_free` commands (`quiet_run` is the only one I'm able to find atm).

Reply:
Won't this be a problem if someone defines a command with an '_' in its name?

Reply:
My idea was to rely on developers using consistent naming style. 
Only other solution I see would be to have decorator parameter which would toggle this replacement on case-by-case basis.

Reply:
It's fine to forbid command names containing '_' but then you need to enforce this otherwise someone is going to be very confused when they define a suite context free command named `my_command` only to have it fail in the absence of a suite.

Reply:
Fixed